clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -1509,7 +1509,149 @@ describe("Consent i18n", () => {
     });
   });
 
-  describe.skip("when localizing privacy_center components", () => {});
+  describe("when localizing privacy_center components", () => {
+    const GEOLOCATION_API_URL = "https://www.example.com/location";
+    const VERIFICATION_CODE = "112358";
+
+    beforeEach(() => {
+      cy.clearAllCookies();
+
+      // Seed local storage with verification data
+      cy.window().then((win) => {
+        win.localStorage.setItem(
+          "consentRequestId",
+          JSON.stringify("consent-request-id")
+        );
+        win.localStorage.setItem(
+          "verificationCode",
+          JSON.stringify(VERIFICATION_CODE)
+        );
+      });
+
+      // Enable GPC
+      cy.on("window:before:load", (win) => {
+        // eslint-disable-next-line no-param-reassign
+        win.navigator.globalPrivacyControl = true;
+      });
+
+      // Intercept sending identity data to the backend to access /consent page
+      cy.intercept(
+        "POST",
+        `${API_URL}/consent-request/consent-request-id/verify`,
+        { fixture: "consent/verify" }
+      ).as("postConsentRequestVerify");
+
+      // Location intercept
+      cy.intercept("GET", GEOLOCATION_API_URL, {
+        fixture: "consent/geolocation.json",
+      }).as("getGeolocation");
+
+      // Experience intercept
+      cy.intercept("GET", `${API_URL}/privacy-experience/*`, {
+        fixture: "consent/experience_privacy_center.json",
+      }).as("getExperience");
+
+      cy.visitWithLanguage("/consent", SPANISH_LOCALE);
+      cy.overrideSettings({ IS_OVERLAY_ENABLED: true });
```

Comment:
One thing you could _try_ to actually get this working as intended would be to change this logic to:
```suggestion
      cy.visitWithLanguage("/", SPANISH_LOCALE);
      cy.overrideSettings({ IS_OVERLAY_ENABLED: true });
      cy.getByTestId({whatever the consent button is}).click();
```
...in other words:
1. Load the home page
2. Override settings to enable overlay
3. Navigate to consent page

That might do the trick?

Reply:
Might work, yeah, I will try it